README.md

```diff
@@ -135,6 +135,19 @@ Then use these bash commands with the above token to create a channel and post:
    -d '{"text": "This is a text post in the test channel", "title": "Test Post", "channel_name": "test_channel"}'
  ```
 
+#### 5c) Configure a sites
+
+Login to the admin ui and create a new site. Some example values for MicroMasters:
+
+ - `key` -  `micromasters`
+ - `title` - `MicroMasters`
+ - `base_url` - `http://localhost:8079/`
+ - `session_url` - `http://localhost:8079/discussionsToken`
+ - `auth_url` - `http://localhost:8079/discussionsToken`
```

Comment:
I don't see an `auth_url` but I do see a `login_url`, is that what you meant?

Reply:
yeah, i just fixed that right before you commented

Reply:
It looks like it's still `auth_url` though, you only corrected the value not the key